### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jysohn23**
Execute update_torch_wheels.sh in current shell to pickup conda

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Follow these steps to train a PyTorch model with Docker on a TPU:
     distribution (only updates your pytorch-nightly conda env), run:
     ```Shell
     (vm)$ cd /usr/share/torch-xla-nightly/pytorch/xla
-    (vm)$ ./scripts/update_nightly_torch_wheels.sh
+    (vm)$ . ./scripts/update_nightly_torch_wheels.sh
```
Comment (dlibenzi) [inline README.md]: Why this change (sourcing it instead of running it)?

Comment (jysohn23) [reply README.md]: Without it the `conda` binary couldn't be found as its sourcing happens in `/etc/profile`.
